src/Bindings/LuaState.cpp

```diff
@@ -39,6 +43,10 @@ extern "C"
 
 const cLuaState::cRet cLuaState::Return = {};
 
+/** Each Lua state stores a pointer to its creating cLuaState in Lua globals, under this name.
+This way any cLuaState can reference the main cLuaState's TrackedCallbacks, mutex etc. */
+static const char * g_CanonLuaStateGlobalName = "__Cuberite_CanonLuaState";
```

Comment:
Both are exported, they need to be - the bindings code needs to read them from the Lua engine in order to know which Plugin / LuaState is currently calling the function.

They should be explicitly ignored in the LuaAPI documentation, so that there's no mention of them in the API docs.

I should probably unify the names, though.